#CREATE request parameters
string[] action
float32 iterations
float32[] angles
--------------------------------
#CREATE response parameters
bool moving